hooks: add hook for easyocr (#530)
pyinstaller/pyinstaller-hooks-contrib: hooks-config.rst

===========================
Supported hooks and options
===========================

> This section lists hooks that implement support for configuration
> options in `pyinstaller-hooks-contrib` repository.

Reviewer: ```suggestion
This section lists hooks that are configurable in `pyinstaller-hooks-contrib`.
```

> Please refer to `Hook Configuration Options <https://pyinstaller.org/en/stable/hooks-config.html>`_

Reviewer: ```suggestion
See also `Hook Configuration Options <https://pyinstaller.org/en/stable/hooks-config.html>`_
```
